scale/storage/brokers/host/host_broker.py

```diff
@@ -0,0 +1,99 @@
+"""Defines a broker that mounts a local host directory into the task container as its backend storage"""
```

Comment:
I think it's more Pythonic to start with just the single module and then create a sub-package if/when you actually need one so you don't have useless directories. See for example Django views.py or tests.py.